web/src/components/console/utils/status-box.tsx

```diff
@@ -20,26 +20,24 @@ const LoadError: React.FC<LoadErrorProps> = ({ label, className, message, canRet
     <Box className={className}>
       <div className="pf-u-text-align-center cos-error-title">
         {_.isString(message)
-          ? t('public~Error Loading {{label}}: {{message}}', {
+          ? t('Error Loading {{label}}: {{message}}', {
               label,
               message,
             })
-          : t('public~Error Loading {{label}}', { label })}
+          : t('Error Loading {{label}}', { label })}
       </div>
       {canRetry && (
         <div className="pf-u-text-align-center">
-          <Trans ns="public">
-            Please{' '}
-            <Button
-              type="button"
-              onClick={window.location.reload.bind(window.location)}
-              variant="link"
-              isInline
-            >
-              try again
-            </Button>
-            .
-          </Trans>
+          {t('Please ')}{' '}
```

Comment:
```suggestion
          {t('Please')}{' '}
```

Reply:
Applied the suggested changes in the latest commit.